### PR TITLE
[New project structure] Adjusting supporting files

### PR DESCRIPTION
**Pull request by bullet-tooth**
## Overview

---
Moved and created supporting files according to the new structure

### Definition of Done

- [ ] There are no TODOs left in the code
- [ ] Change is covered by automated [tests](https://github.com/exonum/exonum-java-binding/blob/master/CONTRIBUTING.md#tests)
- [ ] The [coding guidelines](https://github.com/exonum/exonum-java-binding/blob/master/CONTRIBUTING.md#the-code-style) are followed
- [ ] Public API has Javadoc
- [ ] Method preconditions are checked and documented in the Javadoc of the method
- [ ] Changelog is updated if needed (in case of notable or breaking changes)
- [ ] The continuous integration build passes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,11 @@
...
-  - [Roadmap](ROADMAP.md)
-  - [Contribution Guide](CONTRIBUTING.md)
-  - [Code of Conduct](https://github.com/exonum/exonum/blob/master/CODE_OF_CONDUCT.md)
-
-If you are using Exonum in your project and want to be listed on our website and
-GitHub list — write us a line to <exonum@bitfury.com>.
-
-
+# Exonum Library for Java
+This is a primary repository for Exonum projects for java.
```
Comment (vitvakatu) [inline README.md]:
```suggestion
This is a primary repository for Exonum projects in Java.  
```

Though this is not true. i.e. this repository will not be used for our Java services.

File: README.md [hunk 2]
```diff
@@ -1,28 +1,11 @@
...
-GitHub list — write us a line to <exonum@bitfury.com>.
-
-
+# Exonum Library for Java
+This is a primary repository for Exonum projects for java.  
+The following projects are available:
+ - [Exonum Java Binding](exonum-java-binding-parent)
+
+## Java version
+Java 8 or above is required for working with this repository.
```
Comment (vitvakatu) [inline README.md]:
```suggestion
Java 8 or newer is required for building and using Exonum Java projects.
```

🤔  or smth like this

File: README.md [hunk 3]
```diff
@@ -1,28 +1,11 @@
...
+ 
 ## License
-This project is licensed under the Apache License, Version 2.0.
```
Comment (vitvakatu) [inline README.md]: Why these lines are changed?

File: README.md [hunk 4]
```diff
@@ -1,28 +1,12 @@
-# Exonum Java Binding
-
-[![Build Status](https://img.shields.io/travis/exonum/exonum-java-binding/master.svg)](https://travis-ci.com/exonum/exonum-java-binding)
```
[the remark this answers is not in the document]
Comment (bullet-tooth) [reply README.md]: I suppose we'll have one build for the whole project. And in case of necessity, we can split it in future.

File: README.md [hunk 5]
```diff
@@ -1,28 +1,35 @@
...
+[Exonum][exonum] is an extensible open-source framework for creating blockchain applications.
 
-The project is at alpha stage. Currently you can define a stateful Exonum Service 
-in Java, configure a network of nodes and run the service in it.
+## Projects
+The following projects are available:
+ - [Exonum Java Binding](exonum-java-binding-parent). It is a framework for building blockchain 
+ applications in Java which provides Exonum services runtime environment
+ - [Light Client](exonum-java-light-client). It is a library for interaction with Exonum 
+ on a client site that can be used for creating digital signatures,
```
Comment (MakarovS) [inline README.md]: Did you intend to pit `site` instead of `side` here? It also works but to me `client side` seems more common.

File: README.md [hunk 6]
```diff
@@ -1,28 +1,35 @@
...
+ forming adn sending transactions ect
```
Comment (MakarovS) [inline README.md]: Please fix this line - `and`, `etc.`

Comment (MakarovS) [reply README.md]: Still no dot in the end of the sentence.

File: README.md [hunk 7]
```diff
@@ -5,8 +5,15 @@
...
 
 This is a primary repository for Exonum projects in Java.  
+[Exonum][exonum] is an extensible open-source framework for creating blockchain applications.
+
+## Projects
 The following projects are available:
- - [Exonum Java Binding](exonum-java-binding-parent)
+ - [Exonum Java Binding](exonum-java-binding-parent). It is a framework for building blockchain 
+ applications in Java which provides Exonum services runtime environment
+ - [Light Client](exonum-java-light-client). It is a library for interaction with Exonum
```
Comment (dmitry-timofeev) [inline README.md]: "It is a library for interaction with Exonum (blockchain?) from client (or external?) applications."

I'd consider putting a more detailed _and_ complete description in the corresponding README, not here (digital signatures seem irrelevant).

File: README.md [hunk 8]
```diff
@@ -10,26 +10,28 @@ This is a primary repository for Exonum projects in Java.
 ## Projects
 The following projects are available:
  - [Exonum Java Binding](exonum-java-binding-parent). It is a framework for building blockchain 
- applications in Java which provides Exonum services runtime environment
+ applications in Java which provides Exonum services runtime environment.
  - [Light Client](exonum-java-light-client). It is a library for interaction with Exonum 
- on a client site that can be used for creating digital signatures,
- forming adn sending transactions ect
+  blockchain from external applications built on Java.
```
Comment (dmitry-timofeev) [inline README.md]: from external Java applications?

File: README.md [hunk 9]
```diff
@@ -1,28 +1,37 @@
-# Exonum Java Binding
+# Exonum Library for Java
```
Comment (dmitry-timofeev) [inline README.md]: Exonum Java?
Exonum Java Libraries?

File: README.md [hunk 10]
```diff
@@ -1,28 +1,37 @@
...
-  - [User Guide](https://exonum.com/doc/version/latest/get-started/java-binding/)
-  - [Cryptocurrency Example Service](exonum-java-binding-cryptocurrency-demo)
   - [Roadmap](ROADMAP.md)
-  - [Contribution Guide](CONTRIBUTING.md)
   - [Code of Conduct](https://github.com/exonum/exonum/blob/master/CODE_OF_CONDUCT.md)
 
 If you are using Exonum in your project and want to be listed on our website and
 GitHub list — write us a line to <exonum@bitfury.com>.
 
+## Contributing
```
Comment (dmitry-timofeev) [inline README.md]: @Buzovska , @Maria-Nosyk Shall we put that in a separate section or leave as simple "Contribution Guide"?

Comment (dmitry-timofeev) [reply README.md]: @bullet-tooth, I'd consider moving it back in the list. WDYT?

Comment (bullet-tooth) [reply README.md]: ok
